src/main/mainWindow.ts

```diff
@@ -181,14 +183,34 @@ function initMenuBar(win: BrowserWindow) {
                 app.quit();
             }
         },
-        isDarwin && {
-            label: "Hide",
-            role: "hide"
-        },
-        isDarwin && {
-            label: "Hide others",
-            role: "hideOthers"
-        },
+        ...(!isDarwin ? [] : [
+            {
+                type: "separator"
+            },
+            {
+                label: "Settings",
+                accelerator: "CmdOrCtrl+,",
+                async click() {
+                    mainWin.webContents.executeJavaScript("Vencord.Webpack.Common.SettingsRouter.open()")
```

Comment:
did you even test this? this gives you a blank screen
```suggestion
                    mainWin.webContents.executeJavaScript("Vencord.Webpack.Common.SettingsRouter.open('My Account')")
```

Reply:
Worked fine for me, weird

Reply:
Just realized this seems to open whatever the last page in the settings was, so no wonder it displays a blank screen on the first try.